src/navigation/RootNavigator.tsx

```diff
@@ -11,12 +13,30 @@ import {
   SignInScreen,
   SignUpScreen,
 } from '~screens'
+import { registerForPushNotificationsAsync } from '~services'
 
 const { Navigator, Screen, Group } = createStackNavigator<RootStackParamList>()
 
 export const RootNavigator: FC = () => {
   const { t } = useTranslation()
   const { isSignedIn } = useAuth()
+  const { notification, setNotification } = useNotificationContext()
+
+  useEffect(() => {
+    const initNotifications = async () => {
+      await registerForPushNotificationsAsync()
+    }
+    initNotifications()
+  }, [])
+
+  useEffect(() => {
+    if (notification) {
+      Alert.alert('Notification', JSON.stringify(notification), [
```

Comment:
Maybe instead of displaying it in alert you can use react-native-notificated package that we have implemented.

It's just a proposition, maybe it will look better